src/types/commands.ts

```diff
@@ -98,6 +98,12 @@ export const invalidateEvaluationCommands = new Set<CommandTypes>([
   "INSERT_CELL",
   "UNDO",
   "REDO",
+  "ADD_MERGE",
+]);
+
+export const invalidateDependenciesEvaluationCommands = new Set<CommandTypes>([
+  ...invalidateEvaluationCommands,
+  "PASTE",
```

Comment:
It's specifically the "MOVE_range" command during cut/paste that is the problem no? copy/paste don't change dependencies I think

Reply:
It looks like you're totally right :)